dom0/sd-usb-autoattach-remove.sls

```diff
@@ -0,0 +1,9 @@
+remove-usb-autoattach:
+  cmd.run:
+    - names:
+      - qvm-run sys-usb 'sudo rm -f /etc/udev/rules.d/99-sd-devices.rules'
+      - qvm-run sys-usb 'sudo rm -f /rw/config/sd/etc/udev/rules.d/99-sd-devices.rules'
```

Comment:
nit: why not `rm -rf  /rw/config/sd` here?

Reply:
Just being conservative and only deleting what was added for auto attachment. I can't think of what else we might add to the USB Qube, so it'd probably be fine to remove the whole `sd` directory, but this should be safer. Did I miss something? Is the directory structure being left behind a problem?

Reply:
Fair point on it being safer to delete explicit files, if we do add more files in the future we can re-evaluate.